<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>9</y>
      <width>78</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Encode File</string>
    </property>
   </widget>
   <widget class="QLabel" name="FilePath">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>File Path</string>
    </property>
   </widget>
   <widget class="QLabel" name="SavePath">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>111</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Path</string>
    </property>
   </widget>
   <widget class="QLabel" name="MD5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>164</y>
      <width>33</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MD5</string>
    </property>
   </widget>
   <widget class="QLabel" name="Mode">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>215</y>
      <width>44</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MODE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Mode1radioButton">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>215</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Mode2radioButton">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>215</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="Param">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Param</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="FilePathText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="SavePathText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="MD5Text">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ParmN">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>36</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>n:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ParmNText">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ParmM">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>36</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>m:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ParmMText">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectFileButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectPathButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
